packages/webapp/pages/feeds/[slugOrId]/edit.tsx

```diff
@@ -0,0 +1,234 @@
+
+        onAskConfirmation(false);
+        router.replace('/');
+      },
+    },
+  );
+
+  const shouldRedirectToNewFeed =
+    feeds && feedSlugOrId && deleteStatus === 'idle';
+
+  useEffect(() => {
+    if (!shouldRedirectToNewFeed) {
+      return;
+    }
+
+    if (!feed) {
+      router.push(`${webappUrl}feeds/new`);
+    }
+  }, [shouldRedirectToNewFeed, feed, router]);
+
+  if (!feed) {
+    return null;
+  }
+
+  const seo: NextSeoProps = {
+    title: 'Edit custom feed',
+    openGraph: { ...defaultOpenGraph },
+    ...defaultSeo,
+  };
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <FeedPageLayoutComponent className={classNames('!p-0')}>
+        <LayoutHeader className="flex-col overflow-x-visible">
+          <form
+            className="flex w-full flex-col justify-center bg-gradient-to-b from-surface-invert via-surface-invert"
+            onSubmit={(e) => {
+              e.preventDefault();
+
+              const { name } = formToJson<EditFeedFormProps>(e.currentTarget);
+
+              onSubmit({ name });
+            }}
+          >
+            <FeedCustomActions
+              isDisabled={!isPreviewFeedEnabled}
+              isLoading={isLoading}
+              onDiscard={async () => {
+                const shouldDiscard =
+                  onValidateAction() || (await showPrompt(discardPrompt));
+
+                if (shouldDiscard) {
+                  onAskConfirmation(false);
+
+                  router.push('/');
+                }
+              }}
+              onDelete={onDelete}
+            >
+              <p className="font-bold typo-title3">Feed settings</p>
+            </FeedCustomActions>
+            <TextField
+              className={{
+                container: 'mx-auto mt-10 w-full px-4 tablet:max-w-96',
+              }}
+              defaultValue={feed.flags?.name}
+              name="name"
+              type="text"
+              inputId="feedName"
+              label="Enter feed name"
+              required
+            />
+          </form>
+
+          <div className="flex w-full max-w-full flex-col">
+            <FilterOnboardingV4
+              className="mt-10 px-4 pt-0 tablet:px-10"
+              shouldUpdateAlerts={false}
+              shouldFilterLocally
+              feedId={feedId}
+              onClickTag={() => {
+                setDirty(true);
+              }}
+            />
+            <FeedCustomPreviewControls
+              isOpen={isPreviewFeedVisible}
+              isDisabled={!isPreviewFeedEnabled}
+              onClick={setPreviewFeedVisible}
+            />
+          </div>
+        </LayoutHeader>
+        {isPreviewFeedEnabled && isPreviewFeedVisible && (
+          <FeedCustomPreview feedId={feedId} />
+        )}
+      </FeedPageLayoutComponent>
+    </>
+  );
+};
+
+EditFeedPage.getLayout = getLayout;
+EditFeedPage.layoutProps = {
+  ...mainFeedLayoutProps,
+};
```

Comment:
Think this will do if no overrides are needed
```suggestion
EditFeedPage.layoutProps = mainFeedLayoutProps;
```